sorter.lic

```diff
@@ -0,0 +1,285 @@
+=begin
+
+    This script will display the contents of your containers in categories
+    when you look in them.
+
+    ;sorter width=<window width in characters>    to display contents in columns
+    ;sorter width=nil                             to stop using columns
+
+     author: Tillmen (tillmen@lichproject.org)
+       game: Gemstone
+       tags: cosmetic
+    version: 0.9
+
+    changelog:
+        0.9 (2016-08-17):
+            fix for the last fix
+        0.8 (2016-08-07):
+            fix for containers that give XML of what's inside the container when you look on the container
+        0.7 (2015-04-16):
+            fix display of broken containers on WizardFE
+
+=end
+=begin
+
+        0.6 (2015-02-17):
+            fix the fix for squelching containers that can't be sorted
+        0.5 (2015-02-17):
+            another fix for squelching containers that can't be sorted
+        0.4 (2014-11-21):
+            fix for squelching containers that can't be sorted
+        0.3 (2014-11-21):
+            testing fix for squelching containers that can't be sorted
+        0.2 (2014-11-10):
+            don't display container contents twice on Lich 4.6
+
+=end
+
+custom_require.call(%w[common])
+
+if script.vars[1] =~ /^width=([0-9]+|nil)$/
```

Comment:
💡 Our args system would make this easier to use

Reply:
That option is to do columned sorting and it doesn't usually look right when you do it. I'm not a fan of the look but I left it in because all the work to get it running is already done. Might as well convert it to the arg system.

Reply:
Can add an arg for ignoring sorting groups in base-items.yaml. Currently have it hard-coded to ignore lootables and trash.

Reply:
args added